doc/design.md

```diff
@@ -142,6 +142,17 @@ Covers:
 
 Needs: impl, itest
 
+### ExaLoader in Common Docker Network
+`dsn~exaloader-in-common-docker-network~1`
+
+ETC allows running two test container in the same docker network so that the ExaLoader is able to execute `IMPORT` statements that import data from one database into the other.
```

Comment:
Similarly, I could not understand what `ETC` stands for, I also could not find the expansion on the Readme. Maybe write the expansion for the first time.

Reply:
Exasol Test Containers? ;)

Reply:
Yes, maybe it is better to name it to `two Exasol containers` since this does not work for general containers, I guess.

Reply:
Changed.

Reply:
Added term definition in the introduction section of the document.